CHANGES.rst

```diff
@@ -5,6 +5,7 @@
 1.0.6 (22 Jul 2020)
 -------------------
 
+ - (Hotfix) Deal with chk_calibs crash
```

Comment:
Put this under the new version (1.0.7dev).  I.e., we didn't change the tagged version of the code.  Also, this wasn't a problem with the code, right?  Just the test?

Reply:
fixed